### DISCUSSION TITLE
Fail to upload a product image

### DISCUSSION THREAD

**Discussion by yoan34** [saleor/saleor#9443] [General] [answered]
hi,
I would like to know if any configuration is needed to upload image urls for products, I end up with this error. the url is from backblaze.
URL use: https://s3.us-west-004.backblazeb2.com/saleor-assets/products/983794168_93956210.jpg

![upload-fail](https://user-images.githubusercontent.com/40340706/161260255-a4c8794d-4a8b-4941-8681-d1e48b045f63.png)

**Comment by karolkielecki** [comment, marked as answer]
Hi @yoan34, the Saleor validation library supports the following services: Dailymotion, Flickr, Imgur, Instagram, MobyPicture, Photobucket, SmugMug, Scribd, TikTok, Twitter, Vimeo, WordPress.tv, YouTube.
